src/main/java/zd/zero/waifu/motivator/plugin/listeners/WaifuUnitTester.java

```diff
@@ -1,24 +1,26 @@
-package zd.zero.waifu.motivator.plugin.listeners;
-
-import com.intellij.openapi.project.Project;
-
-public interface WaifuUnitTester {
-
-    static WaifuUnitTester newInstance( Project project ) {
-        return new WaifuUnitTesterImpl( project.getMessageBus().connect(),
-                new WaifuUnitTesterListenerImpl( project ) );
-    }
-
-    void init();
-
-    void stop();
-
-    interface Listener {
-
-        void onUnitTestPassed();
-
-        void onUnitTestFailed();
-
-    }
-
-}
+package zd.zero.waifu.motivator.plugin.listeners;
+
+import com.intellij.openapi.project.Project;
+
+public interface WaifuUnitTester {
+
+    static WaifuUnitTester newInstance( Project project ) {
+        return new WaifuUnitTesterImpl( project.getMessageBus().connect(),
+                new WaifuUnitTesterListenerImpl( project ),
+            new JetbrainsDebouncer( 250 )
```

Comment:
minor: Hmmm, I suggest if we extract the magic number `250` to a local variable with `final int INTERVAL` so readers don't need to check the constructor signature for its usage.

Reply:
I can change that.

You're IDE does not show the parameter hints?
![image](https://user-images.githubusercontent.com/15972415/88464784-25e4a300-ce83-11ea-8809-f3fe2dc2ef04.png)

Reply:
I also made up 250, it could probably be shorter.

Reply:
Parameter hints are displayed on my IDE, but for others where it is not configured or especially when just glancing through GH PR directly.